### ISSUE TITLE
The repository has moved!!! https://github.com/evolution-cms/evolution/

### ISSUE THREAD

**Issue by dmi3yy** [modxcms/evolution#1392]
New home of Evolution here: https://github.com/evolution-cms/evolution/
New questions please write there: https://github.com/evolution-cms/evolution/issues

**Comment by janniconl** [comment]
Sorry, may I ask why it is moved? I may have missed a discussion here, if so, can you point me there?

What will be done with open issues in this git? Forgotten?

**Comment by Deesen** [comment]
Maybe this post helps?

https://forums.modx.com/thread/102566/important-modx-evo-is-now-evolution-cms

**Comment by janniconl** [comment]
Yes, super, thx! (saves me some explaining to some of my customers :)

**Comment by dmi3yy** [comment]
modx.com/blog/evolution-cms-has-a-new-home
